src/main/java/com/slack/eithernet/ApiResult.kt

```diff
@@ -308,6 +308,15 @@ public object ApiResultCallAdapterFactory : CallAdapter.Factory() {
                   val withTag =
                     when (val result = response.body()) {
                       is Success -> result.withTags(result.tags + tags)
+                      null -> {
+                        val successType = apiResultType.actualTypeArguments[0]
+                        val responseCode = response.code()
+                        if ((responseCode == 204 || responseCode == 205) && successType == Unit::class.java) {
+                          ApiResult.success(Unit).withTags(emptyMap<KClass<*>, Any>() + tags)
+                        } else {
+                          null
```

Comment:
if this is going to NPE later, I suspect we'll want to throw a more useful error message. I can take a look at that in a followup though

Reply:
Didn't want to, since this actually ends up crashing on a retrofit method and so they might end up changing that and then we would be throwing an error message and it could end up not crashing. I can leave a comment, but let me know what you think

Reply:
Don't worry about it, I'll look in on my own